src/api-service/__app__/timer_daily/__init__.py

```diff
@@ -7,15 +7,30 @@
 
 import azure.functions as func
 from onefuzztypes.enums import VmState
+from onefuzztypes.events import EventProxyCreated
 
-from ..onefuzzlib.events import get_events
+from ..onefuzzlib.events import get_events, send_event
 from ..onefuzzlib.proxy import Proxy
 from ..onefuzzlib.webhooks import WebhookMessageLog
 from ..onefuzzlib.workers.scalesets import Scaleset
 
 
 def main(mytimer: func.TimerRequest, dashboard: func.Out[str]) -> None:  # noqa: F841
     for proxy in Proxy.search():
+        if proxy.is_outdated():
```

Comment:
In `is_outdated`, you set proxy.outdated, as well as setting it 2 lines later.

I would not expect a function `is_<phrase>` to have side effects.  As such, I think is clearer to keep the `proxy.outdated = True` in this function and to remove it in `is_outdated`.

Reply:
Good point. Made change.